addons/l10n_ae/views/report_invoice_templates.xml

```diff
@@ -1,6 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <template id="l10n_ae_report_invoice_document" inherit_id="l10n_gcc_invoice.l10n_gcc_report_invoice_document" primary="True">
+        <!-- Titles -->
+        <t name="credit_note_title" position="replace">
+            <t name="credit_note_title">Tax Credit Note</t>
+        </t>
+        <t name="credit_note_title_ar" position="replace">
+            <t name="credit_note_title_ar">إشعار دائن ضريبي</t>
+        </t>
+        <t name="credit_note_title_en" position="replace">
+            <t name="credit_note_title_en">Tax Credit Note</t>
```

Comment:
Just a question why do we need credit_note_title and credit_note_title_en they are suppose to be the same since we are suppose to write every template in english ?